.dockerignore

```diff
@@ -0,0 +1,8 @@
+app/etc/config.php
+app/etc/env.php
+vendor
+generated
+pub/static
+pub/media
+var/view_preprocessed
+.git
```

Comment:
I believe `.dockerignore` is only useful when building images, which should be done from the `cloud-docker` repository, so this file shouldn't belong here

Reply:
Agree, removed